src/qt/bitcoin.cpp

```diff
@@ -352,6 +352,17 @@ void BitcoinApplication::requestShutdown()
     window->setClientModel(nullptr);
     pollShutdownTimer->stop();
 
+#ifdef ENABLE_WALLET
+    // Delete wallet controller here manually, instead of relying on Qt
+    // reference counting. This makes sure walletmodel m_handle_* notification
```

Comment:
re: https://github.com/bitcoin/bitcoin/pull/19101#discussion_r646927022

> [fe45c37](https://github.com/bitcoin/bitcoin/commit/fe45c379898e1a418af3847974a55f936814ca2a)
> 
> > instead of relying on Qt reference counting.
> 
> What Qt reference counting?

Thanks, changed "reference counting" to "object tracking" and linked to https://doc.qt.io/qt-5/objecttrees.html